ui/app/AppLayouts/Chat/ChatColumn/ChatInput.qml

```diff
@@ -31,11 +32,12 @@ Rectangle {
 
     function onEnter(event){
         if (event.modifiers === Qt.NoModifier && (event.key === Qt.Key_Enter || event.key === Qt.Key_Return)) {
-            if(txtData.text.trim().length > 0){
-                let msg = interpretMessage(txtData.text.trim())
+            let rawText = txtData.text.replace(/<[^>]+>/g, '').replace('p, li { white-space: pre-wrap; }', '').trim()
```

Comment:
You use those replaces a lot. Might be worth putting it in `imports/Utils.qml` as a function

Reply:
Ah, didn't know about that one!